hydra/hydra.yml

```diff
@@ -5,6 +5,7 @@ serve:
     cors:
       enabled: true
       allowed_origins:
+        - https://oauthdebugger.com
         - http://localhost:10000
```

Comment:
Can you use environment variables to setup this ? It's easier for defining values for our environments

Reply:
It should not be used outside of development purpose.
I don't know if env variable are workig in this yaml with Hydra. 
I could just allow everything to simplify.